### PR TITLE
feat: Added missing keys for container definition

### LINKED ISSUE #86: Missing keys
container definition docs: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definition_parameters.html#advanced_container_definition_params

* hostname
* disableNetworking
* dockerSecurityOptions
* interactive
* pseudoTerminal

There will be more in the future. At least any missing keys can be added using the `var.container_definition` override.

### PR DESCRIPTION
**Pull request by zahorniak**
## what
Added missing keys for container definition:

- hostname
- disableNetworking
- dockerSecurityOptions
- interactive
- pseudoTerminal

## why
These changes should simplify work with this module. People no longer need to use `container_definition` variable to override options.

## references
Closes #86

### CODE REVIEW

**Comment by zahorniak** [comment]
@nitrocode can you merge this pr?

File: examples/complete/versions.tf [hunk 1]
```diff
@@ -1,5 +1,5 @@
 terraform {
-  required_version = "~> 0.12.0"
+  required_version = ">= 0.12.0, < 0.14"
```
Comment (osterman) [inline examples/complete/versions.tf:2]:
```suggestion
  required_version = ">= 0.12.0"
```

**Review by rb-nr — APPROVED** [review]

**Review by nitrocode — APPROVED** [review]

**Pull request merged**